src/main/java/core/basesyntax/util/FileWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.util;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void write(List<String> content, String filePath) {
+        try {
+            Files.write(Path.of(filePath), content);
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot write file by path " + filePath);
```

Comment:
The exception message in the catch block should include more details about the exception to aid in debugging. Consider logging the exception or providing more context in the error message.